include/gimp_element.h

```diff
@@ -31,19 +32,21 @@ namespace mpm {
 //!   |           |           |           |
 //!   |           |           |           |
 //!   4-----------5-----------6-----------7
+//!
 //! </pre>
 //! \tparam Tdim Dimension
 //! \tparam Tnfunctions Number of functions
-template <unsigned Tdim, unsigned Tnfunctions>
-class GimpElement : public Element<Tdim> {
+
+class QuadrilateralGIMPElement
+    : public QuadrilateralElement<Tdim, Tnfunctions> {
 
```

Comment:
@kks32  Here is where I am having syntax trouble, as Tdim and Tnfunctions aren't declared.